cmd/app-mesh-inject/main.go

```diff
@@ -106,6 +109,11 @@ func main() {
 		}
 	}
 
+	// Only one tracer is supported
+	if cfg.EnableDatadogTracing && cfg.EnableJaegerTracing {
```

Comment:
This should include checking xray too.

Reply:
Thanks for bringing that up! I am adding a method to check that only one tracer is on.